db/migrate/20161205144737_add_content_ids_to_needs.rb

```diff
@@ -0,0 +1,10 @@
+class AddContentIdsToNeeds < Mongoid::Migration
+  def self.up
+    Need.all.each do |n|
+      unless n.content_id.present?
+        n.content_id = SecureRandom.uuid
+        puts "Set content_id #{n.content_id} for #{n.id}."
```

Comment:
This output was pretty noisy in dev - should we keep it? It'll only appear in the console when the migration is running, and it's generally useful to know that things are doing what you think they are, but there are a lot of needs.